frontend/src/navbar.css

```diff
@@ -0,0 +1,46 @@
+html, body {
+    margin: 0;
+    background-color: rgb(249, 254, 255);
+    max-width: 100%;
+    overflow-x: hidden;
+    font-family: Arial, Helvetica, sans-serif;
+   }
+
+header {
+    background-color : rgb(49, 221, 150);
+    width : 100%;
+    display: flex;
+    flex-direction: row;
+}
+.topnav {
+    text-align: center;
+    width: 50%;
+    padding : 50px;
+}
+.topnav a {
+    text-align: center;
+    padding: 10px;
+    font-size: 25px;
+    font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
I'd recommend adding a property so your links don't wrap between lines. There are many ways to do this in CSS, but the `display` property could be a good place to start.
![image](https://user-images.githubusercontent.com/32073638/142077506-d4ef6b79-ab2e-4612-9cd8-728dc8225539.png)